modules/wsdata_block/src/Plugin/Block/WSDataBlock.php

```diff
@@ -67,7 +67,7 @@ public function blockSubmit($form, FormStateInterface $form_state) {
   public function build() {
     $form = [];
     $wsdata  = \Drupal::service('wsdata');
-    $result = $wsdata->call($this->configuration['wscall'], NULL, $this->configuration['replacements'], $this->configuration['data'], array(), $this->configuration['returnToken']);
+    $result = $wsdata->call($this->configuration['wscall'], NULL, $this->configuration['replacements'], $this->configuration['data'], [], $this->configuration['returnToken']);
```

Comment:
If that array is an option argument would it be better as a NULL or at the end of the list?

Reply:
I'd rather leave is as an array as a hint to anyone working on this code in the future.